### PR TITLE
Rename ec2_vpc_peer to ec2_vpc_peering

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Rename ec2_vpc_peer to ec2_vpc_peering

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: meta/runtime.yml [hunk 1]
```diff
@@ -189,6 +190,13 @@ plugin_routing:
         warning_text: >-
           rds_param_group has been renamed to rds_instance_param_group.
           Please update your tasks.
+    ec2_vpc_peer:
+      redirect: amazon.aws.ec2_vpc_peer
+      deprecation:
+        removal_version: 11.0.0
```
Comment (tremble) [inline meta/runtime.yml:196]: Is 1 year long enough?

I would be inclined to push this out to 13, and start the deprecation of aliases from the mass-rename...

**Review by tremble — APPROVED** [review]
I'm good with the actual rename.  I'd rather the deprecation cycle was a little longer, but I wouldn't block on this.

File: meta/runtime.yml (hunk 1 above)
Comment (alinabuzachis) [reply meta/runtime.yml]: Updated.

**Comment by alinabuzachis** [comment]
recheck

**Pull request merged**